### DISCUSSION TITLE
where is the mods

### DISCUSSION THREAD

**Discussion by laenaneeds** [thatskymod/Sky-CotL-Scripts#131] [Q&A] [answered]
![Screenshot 2024-09-06 210239](https://github.com/user-attachments/assets/a4e0dbca-82db-4248-994f-4c31afb959e1)

**Comment by thatskymod** [comment, marked as answer]
Hello,

you might want to try the [ThatSkyMod](https://github.com/XeTrinityz/ThatSkyApp/) installer by @XeTrinityz, which would streamline the process of downloading and installing mods. The installer automatically places everything in the correct location. Using this tool should resolve your issue with the 0 mods showing up.

Let me know if this helps!
